CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 ## [x.x.x]
 ### Added
 ### Fixed
+- Freeze coloredlogs temporarily
```

Comment:
Famous last words..

Reply:
🤣 I know. I felt it as soon as I hit commit.